app/parsers/bulkrax/csv_parser.rb

```diff
@@ -74,8 +74,22 @@ def create_collections
       collections.each_with_index do |collection, index|
         next if collection.blank?
         break if records.find_index(collection).present? && limit_reached?(limit, records.find_index(collection))
+        ActiveSupport::Deprecation.warn(
+          'Creating Collections using the collection_field_mapping will no longer be supported as of Bulkrax version 3.0.' \
+          ' Please configure Bulkrax to use related_parents_field_mapping and related_children_field_mapping instead.'
+        )
+
+        metadata = if collection.key?(:title) && collection.keys.count == 1
```

Comment:
@ShanaLMoore can we add a comment above this explaining why it's here and that it should be removed when `collection_field_mapping` is removed please?